tmp/lanelet2_extension/lib/query.cpp

```diff
@@ -879,6 +879,19 @@ std::vector<lanelet::ConstLanelets> query::getPrecedingLaneletSequences(
   return lanelet_sequences_vec;
 }
 
+lanelet::ConstPolygons3d query::getAllNoDetectionArea(
+  const lanelet::LaneletMapConstPtr & lanelet_map_ptr)
+{
+  lanelet::ConstPolygons3d no_detection_area;
+  for (const auto & poly : lanelet_map_ptr->polygonLayer) {
+    const std::string type = poly.attributeOr(lanelet::AttributeName::Type, "none");
+    if (type == "no_detection_area") {
+      no_detection_area.push_back(poly);
+    }
+  }
+  return no_detection_area;
+}
```

Comment:
[minor comment]
Better to put this in the same position as hpp.

Reply:
fixed the position of the function.
https://github.com/autowarefoundation/autoware_common/pull/92/commits/99feb65928eb3d9bda1629909d023e005b9620f7